src/rpc/scripthash.rs

```diff
@@ -60,9 +60,16 @@ pub fn get_first_use(query: &Query, scripthash: &FullHash) -> Result<Value> {
         }
     };
 
+    let height = if firstuse.0 == MEMPOOL_HEIGHT {
+        0
+    } else {
+        firstuse.0
+    };
```

Comment:
the latest forced push is just linting right?

Reply:
Yes :-)

Reply:
Travis is broken and didn't catch it, migrating to Github CI in #84